package.json

```diff
@@ -68,7 +68,7 @@
     "ethers": "~5.0.8",
     "husky": "~4.2.5",
     "lint-staged": "~10.2.2",
-    "pegasys-orchestrate": "~3.1.0",
+    "pegasys-orchestrate": "^3.1.2",
```

Comment:
not better to fix the patch version?

Reply:
you are right, in order to align with rest of dep elements